schemas/com.snowplowanalytics.datamodeling/config/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+                        "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$",
+                        "maxLength": 256
+                },
+                "data": {
+                        "type": "object",
+                        "properties": {
+                                "enabled": {
+                                        "type": "boolean"
+                                },
+                                "lockType": {
+                                        "type": "string",
+                                        "enum": [
+                                                "hard",
+                                                "soft"
+                                        ]
+                                },
+                                "storage": {
+                                        "type": "string",
+                                        "enum": [
+                                                "Default",
+                                                "BigQuery"
+                                        ]
+                                },
+                                "playbooks": {
+                                        "type": "array",
+                                        "items": {
+                                                "type": "object",
+                                                "properties": {
+                                                        "playbook": {
+                                                                "type": "string",
+                                                                "minLength": 1,
+                                                                "maxLength": 64
+                                                        },
+                                                        "dependsOn": {
+                                                                "type": "array",
+                                                                "items": {
+                                                                        "type": "string",
+                                                                        "minLength": 1,
+                                                                        "maxLength": 64
+                                                                },
+                                                                "minItems": 0,
+                                                                "maxItems": 64
+                                                        }
+                                                },
+                                                "required": [
+                                                        "playbook",
+                                                        "dependsOn"
+                                                ],
+                                                "additionalProperties": false
```

Comment:
I would probably say additionalProperties = true. 

additionalProperties means that if false, sending additional keys will render the data invalid. If true, additional data can be present, but nothing will be done with it. I think the latter is a better descriptor of the situation (unless your validation script imposes it?)